src/Type/Prelude.purs

```diff
@@ -1,5 +1,11 @@
 module Type.Prelude
-  ( module Proxy
+  ( module Equality
+  , module Data.Ordering
+  , module Proxy
+  , module Data.Symbol
```

Comment:
Minor thing, but we don't need to alias modules for export, if you just have

```
import Type.Equality (class TypeEquals, from, to)
```

then exporting `Type.Equality` works, and only exports the imported members.